assets/src/edit-story/components/library/mediaLibrary.js

```diff
@@ -171,14 +170,12 @@ function MediaLibrary( { onInsert } ) {
 	const onFilter = ( filter ) => {
 		if ( filter !== mediaType ) {
 			setMediaType( filter );
-			setIsMediaLoading( false );
-			setIsMediaLoaded( false );
+			reloadMedia();
 		}
 	};
 
 	const onClose = () => {
-		setIsMediaLoading( false );
-		setIsMediaLoaded( false );
+		resetMedia();
 	};
```

Comment:
This is now just: `const onClose = resetMedia;`.

But don't we want to memoize the callbacks here with `useCallback`? It's not always strictly necessary, but I find it a good practice nevertheless - to make sure not forget where it is needed (to avoid infinite render loops).

Reply:
Use callback. https://github.com/google/web-stories-wp/pull/210/commits/efceca53f136988b665cbd268d9d5177cf9178af